src/main/java/uk/gov/pay/ledger/report/dao/PerformanceReportDao.java

```diff
@@ -41,4 +60,13 @@ public PerformanceReportEntity performanceReportForPaymentTransactions(Performan
             return query.map(new PerformanceReportEntityMapper()).findOnly();
         });
     }
+
+    public List<GatewayAccountMonthlyPerformanceReportEntity> monthlyPerformanceReportForGatewayAccounts(ZonedDateTime startDate, ZonedDateTime endDate) {
+        return jdbi.withHandle(handle ->
+                handle
+                        .createQuery(MONTHLY_GATEWAY_ACCOUNT_PERFORMANCE_STATISTICS)
+                        .bind("startDate", startDate) // 2019-01-01
+                        .bind("endDate", endDate) // 2019-02-01
```

Comment:
can we get rid of the comments in the above two lines please

Reply:
Done.